api/route/campaign_route.py

```diff
@@ -1,7 +1,10 @@
 from flask import Blueprint, request
+from flask_sqlalchemy import SQLAlchemy
 
+from api.model.Campaign import CampaignModel
```

Comment:
are we using `CampaignModel` in this file? if not the import is unnecessary right?